.github/workflows/reuse-run-e2e-tests.yml

```diff
@@ -64,7 +64,7 @@ jobs:
   # Required job
   run-e2e-tests:
     # We can only use conditionals, and not path filters to 'successfully' skip a required job - https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks
-    if: ${{ inputs.has-changes-requiring-build == 'true' }}
+    if: ${{ inputs.e2e-toggle == 'true' }}
```

Comment:
my only comment is that using something like `e2e-toggle` is less descriptive than let's say `has-changes-requiring-e2e-testing`

Reply:
fixed